.changeset/warm-ducks-shout.md

```diff
@@ -0,0 +1,6 @@
+---
+'@leafygreen-ui/menu': major
+'@leafygreen-ui/mongo-nav': minor
+---
+
+MongoNav menus are now scrollable instead of cut-off when window height is too short. Menu is now scrollable when contents exceeds its size. Previously the excess content would be hidden.
```

Comment:
nit: prefer to separate these into two changesets so that the mongo nav updates don't appear in the menu changelog